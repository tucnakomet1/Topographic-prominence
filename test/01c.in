47 42 47 43 48 48 42
