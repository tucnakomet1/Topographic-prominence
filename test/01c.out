5
4
48
